reggie/configs/data/washington.yaml

```diff
@@ -20,6 +20,8 @@ voter_status_inactive: I
 democratic_party: null
 republican_party: null
 no_party_affiliation: null
+abbreviation: wa
+precinct_identifer: CountyCode
```

Comment:
typo identif*I*er - and this should probably be set to "PrecinctCode" right? (Precinct_identifier isn't used in VS right now, but it will be in the future, and also for BallotShield.)